python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -287,8 +291,18 @@
 """
 
 
-def _create_run(graphql_context, pipeline_name, mode="default", step_keys=None):
-    selector = infer_pipeline_selector(graphql_context, pipeline_name)
+def _create_run(
+    graphql_context, pipeline_name, mode="default", step_keys=None, asset_selection=None
+):
+    if asset_selection:
+        selector = infer_job_or_pipeline_selector(
+            graphql_context, pipeline_name, asset_selection=asset_selection
+        )
+    else:
+        selector = infer_pipeline_selector(
+            graphql_context,
+            pipeline_name,
+        )
```

Comment:
nit: not totally obvious to me why these are split [1]

Reply:
They represent different graphql classes
- job_or_pipeline_selector maps to GrapheneJobOrPipelineSelector, which is responsible for execution job subsets
- pipeline_selector maps to GraphenePipelineSelector, which is responsible for fetching pipeline snapshots/execution plan